### ISSUE TITLE
Please add ... to the formal arguments of tidycensus:::use_tigris()?

### ISSUE THREAD

**Issue by NikKrieger** [walkerke/tidycensus#135]
I've been working on a function that calls `get_acs()`/`get_decennial()`. These two have `...` in their formal arguments, allowing them to accommodate other arguments, but then these functions pass their `...` onto `tidycensus:::use_tigris()`.

Since `tidycensus:::use_tigris()` doesn't have `...` in its formal arguments, if you include any arguments in `get_acs()`/`get_decennial()` that aren't in the formal arguments of `tidycensus:::use_tigris()`, an error is thrown.

If you add `...` to the formal arguments of `tidycensus:::use_tigris()`, performance would be the same and an error would not be thrown.

I'm very open to hearing a good reason not to do this.

Thanks!

**Comment by NikKrieger** [comment]
One more thing: I acknowledge that this is only a problem when setting `geometry = TRUE` in `get_acs()`/`get_decennial` because `tidycensus:::use_tigris()` isn't called when `geometry = FALSE`.

**Comment by walkerke** [comment]
@NikKrieger yes I'll do that.  I'll be prepping tidycensus for a new release soon and this will be in there.

**Comment by NikKrieger** [comment]
Awesome. Thank you.

**Comment by NikKrieger** [comment]
Nothing specific at this point. I just don't want users to get an error if they pass an extra non formal argument to `get_acs()`.
